subprojects/docs/src/samples/native-binaries/cpp-exe/build.gradle

```diff
@@ -20,7 +20,7 @@ model {
                 dependsOn binary.tasks.link
                 doFirst {
                     if (binary.toolChain in Gcc) {
-                        ["strip", binary.tasks.link.outputFile].execute()
+                        ["strip", binary.tasks.link.outputFile].execute().waitFor()
```

Comment:
In Groovy we'd use this:

http://docs.groovy-lang.org/latest/html/api/org/codehaus/groovy/runtime/ProcessGroovyMethods.html#waitForOrKill-java.lang.Process-long-